app/api/dao/task.py

```diff
@@ -136,7 +136,8 @@ def complete_task(user_id: int, mentorship_relation_id: int, task_id: int):
 
         if not (user_id == relation.mentee_id or user_id == relation.mentor_id):
             return messages.USER_NOT_INVOLVED_IN_THIS_MENTOR_RELATION, HTTPStatus.UNAUTHORIZED
-
+        if relation.state != MentorshipRelationState.ACCEPTED:
+            return messages.UNACCEPTED_STATE_RELATION, HTTPStatus.BAD_REQUEST
```

Comment:
add a blank line below please, just to separate visually each validation section

Reply:
I added the blank spaces!